lighting.py

```diff
@@ -69,6 +72,9 @@ def set_hue(hue: int):
     global dev
     global dev_state
 
+    # convert bytes to little-endian unsigned integer
+    hue = struct.unpack("<I", hue)[0]
+
```

Comment:
What is this for? 
The input is an integer: https://python-reference.readthedocs.io/en/latest/docs/ints/